geode-core/src/main/java/org/apache/geode/Delta.java

```diff
@@ -53,7 +51,27 @@
    * This method throws an {@link InvalidDeltaException} when the delta in the {@link DataInput}
    * cannot be applied to the object. GemFire automatically handles an {@link InvalidDeltaException}
    * by reattempting the update by sending the full application object.
-   *
    */
   void fromDelta(DataInput in) throws IOException, InvalidDeltaException;
+
+  /**
```

Comment:
The JDK classes use a single unclosed `<p>` at the start of each paragraph in javadocs:
```
/**
   * By default, entry sizes are not recalculated when deltas are applied. This optimizes for the
   * case where the size of an entry does not change. However, if an entry size does increase or
   * decrease, this default behavior can result in the memory usage statistics becoming inaccurate.
   * These are used to monitor the health of Geode instances, and for balancing memory usage across
   * partitioned regions.
   *
   * <p>
   * There is a system property, gemfire.DELTAS_RECALCULATE_SIZE, which can be used to cause all
   * deltas to trigger entry size recalculation when deltas are applied. By default, this is set
   * to 'false' because of potential performance impacts when every delta triggers a recalculation.
   *
   * <p>
   * To allow entry size recalculation on a per-delta basis, classes that extend the Delta interface
   * should override this method to return 'true'. This may impact performance of specific delta
   * types, but will not globally affect the performance of other Geode delta operations.
   *
   * @since 1.14
   */
 ```

Reply:
Just to be sure... do we really want to document `gemfire.DELTAS_RECALCULATE_SIZE` in the javadocs of a User API class? Is it new? I think new system properties should be documented as `geode.PROPERTY` instead of `gemfire.`.

Reply:
gemfire.DELTAS_RECALCULATE_SIZE is not new; it's been around since at least 2017 (git-annotate shows "4/24/17 Lund") and its definition was tweaked "12/11/19 Lund". (Though that was a large commit touching a lot of files.)

It probably makes sense to document it somewhere else, too, but it seems relevant here...

Reply:
(The <p> tag has been addressed.)